pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -113,6 +120,11 @@ type GCECloud struct {
 	// lock to prevent shared resources from being prematurely deleted while the operation is
 	// in progress.
 	sharedResourceLock sync.Mutex
+	// alphaAPI includes all gce alpha api that has been enabled in GCECloud instance.
+	// Related wrapper functions that interacts with gce alpha api should examine whether
+	// the corresponding api is enabled.
+	// If not enabled, it should return error.
+	alphaAPI map[string]bool
```

Comment:
Suggestion: move the relevant code to a separate file with more helper methods e.g., `Enabled(feature string)`. 
This would make writing unit tests for reading the list of features (line 258-274) easier as well.

It'd also be easier to track the status and progress of the alpha features.